pkg/snap/snap/snapcraft.yaml

```diff
@@ -0,0 +1,76 @@
+name: solvespace
+base: core18
+summary: Parametric 2d/3d CAD
+adopt-info: solvespace
+description: |
+  SOLVESPACE is a free (GPLv3) parametric 3d CAD tool.
+  Applications include
+  * modeling 3d parts — draw with extrudes, revolves, and Boolean (union / difference) operations
+  * modeling 2d parts — draw the part as a single section, and export DXF, PDF, SVG; use 3d assembly to verify fit
+  * 3d-printed parts — export the STL or other triangle mesh expected by most 3d printers
+  * preparing CAM data — export 2d vector art for a waterjet machine or laser cutter; or generate STEP or STL, for import into third-party CAM software for machining
+  * mechanism design — use the constraint solver to simulate planar or spatial linkages, with pin, ball, or slide joints
+  * plane and solid geometry — replace hand-solved trigonometry and spreadsheets with a live dimensioned drawing
+
+grade: stable
+confinement: strict
+
+layout:
+  /usr/share/solvespace:
+    bind: $SNAP/usr/share/solvespace
+
+apps:
+  solvespace:
+    command: usr/bin/solvespace
+    extensions: [gnome-3-28]
+    plugs: [opengl, unity7, home, removable-media, gsettings, network]
```

Comment:
`unity7`?

Reply:
From: https://snapcraft.io/docs/unity7-interface I quote:

> Condition to Connect: If your snap has a graphical user interface and is expected to be launched on Unity desktop environment(the default DE on Ubuntu 16.04), you should connect this interface.

And https://snapcraft.io/docs/desktop-interfaces tells us
> the unity7 interface remains for compatibility for typical, non-Wayland capable DEs

and explains its possible use for a
> Desktop app that can run anywhere

So probably a good idea to keep around for maximum compatibility with all sorts of desktop environments

Reply:
Strange name, but OK.